src/lib/sql/sql_identifier_resolver.hpp

```diff
@@ -25,9 +25,11 @@ class SQLIdentifierResolver final {
  public:
   /**
    * @{
-   * Set/Update the column/table name of an expression
+   * Set/Update/Delete the column/table names of an expression. There can be multiple column names because a new alias
```

Comment:
"There can be multiple column names *referring to a single expression*"?

Reply:
Good catch. It should make more sense now.

Reply:
I'd expect it to be a suggestion